static/app/views/replays/filters.tsx

```diff
@@ -47,3 +48,88 @@ const FilterContainer = styled('div')`
 `;
 
 export default ReplaysFilters;
+
+const OMITTED_TAGS = [
+  'culprit',
+  'device.arch',
+  'device.battery_level',
+  'device.charging',
+  'device.locale',
+  'device.online',
+  'device.orientation',
+  'device.simulator',
+  'device.uuid',
+  'error',
+  'error.handled',
+  'error.mechanism',
+  'error.type',
+  'error.unhandled',
+  'error.value',
+  'event.type',
+  'geo',
+  'geo.city',
+  'geo.country_code',
+  'geo.region',
+  'has',
+  'http',
+  'http.method',
+  'http.referer',
+  'http.url',
+  'id',
+  'issue',
+  'level',
+  'location',
+  'measurements',
+  'measurements.app_start_cold',
+  'measurements.app_start_warm',
+  'measurements.cls',
+  'measurements.fcp',
+  'measurements.fid',
+  'measurements.fp',
+  'measurements.frames_frozen',
+  'measurements.frames_frozen_rate',
+  'measurements.frames_slow',
+  'measurements.frames_slow_rate',
+  'measurements.frames_total',
+  'measurements.lcp',
+  'measurements.stall_count',
+  'measurements.stall_longest_time',
+  'measurements.stall_percentage',
+  'measurements.stall_total_time',
+  'measurements.ttfb',
+  'measurements.ttfb.requesttime',
+  'message',
+  'platform',
+  'project',
+  'release.build',
+  'release.package',
+  'release.stage',
+  'release.version',
+  'timestamp',
+  'timestamp.to_day',
+  'timestamp.to_hour',
+  'spans',
+  'spans.browser',
+  'spans.db',
+  'spans.http',
+  'spans.resource',
+  'spans.ui',
+  'stack',
+  'stack.abs_path',
+  'stack.colno',
+  'stack.filename',
+  'stack.function',
+  'stack.in_app',
+  'stack.lineno',
+  'stack.module',
+  'stack.package',
+  'stack.stack_level',
+  'title',
+  'trace',
+  'trace.parent_span',
+  'trace.span',
+  'transaction',
+  'transaction.duration',
+  'transaction.op',
+  'transaction.status',
+];
```

Comment:
this list feels fragile, if someone adds something to the base list and we don't omit it then we'll have a problem.

Also, it looks like fields we should support are not part of the base list, so they're not in the typeahead at all